berlin-model-source/attributes-value_sets.csv

```diff
@@ -0,0 +1,26 @@
+Clinical finding SCTID,Clinical finding term,Attribute SCTID or custom ID,Attribute term,Multiselect,Value SCTID or custom ID,Value term

```

Comment:
I believe the content of the `berlin-model-source` folder are already within https://github.com/FG-AI4H-TG-Symptom/fgai4h-tg-symptom-models-schemas. Maybe we should use that repo as a git submodule? cc: @mzwiessele

Reply:
@adampbaker did you just the files, or are they different from the ones Lucas pointed out?

Reply:
The source data is slightly different to what is in https://github.com/FG-AI4H-TG-Symptom/fgai4h-tg-symptom-models-schemas to make the job of parsing it easier. The csv files are derived from a new spreadsheet which combines the data from the [berlin model definition](https://docs.google.com/spreadsheets/d/111D40yoJqvvHZEYI8RNSnemGf0abC9hQjQ7crFzNrdk) and the [helpers sheets](https://docs.google.com/spreadsheets/d/1fbFNh-vpqlIljIg2OMlXmLCbqeV9ISL4eRaepW0jmjo) used by the schemas. If we need to update the content of the Berlin model based on this spreadsheet from which these csvs are derived then I will add a link to this